save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/filters/OrganizationFilters.kt

```diff
@@ -10,9 +10,10 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class OrganizationFilters(
     val prefix: String,
-    val status: OrganizationStatus = OrganizationStatus.CREATED,
+    val status: List<OrganizationStatus> = listOf(OrganizationStatus.CREATED),
```

Comment:
suggestion to use `set` instead of `list` here -- here is not required ordering and duplicate values are not meaningful in filtering

Reply:
Done